Plugins/Public/playercntl_plugin/MiscCmds.cpp

```diff
@@ -252,6 +263,185 @@ namespace MiscCmds
 		return true;
 	}
 
+	void MiscCmds::LoadListOfReps()
+	{
+		INI_Reader ini;
+
+		string factionpropfile = "..\\data\\initialworld.ini";
+		if (ini.open(factionpropfile.c_str(), false))
+		{
+			while (ini.read_header())
+			{
+				if (ini.is_header("Group"))
+				{
+					uint ids_name;
+					string nickname;
+					while (ini.read_value())
+					{
+						if (ini.is_value("nickname"))
+						{
+							nickname = ini.get_value_string();
+						}
+						else if (ini.is_value("ids_name"))
+						{
+							ids_name = ini.get_value_int(0);
+						}
+
+					}
+					factions[nickname] = ids_name;
+				}
+			}
+			ini.close();
+			ConPrint(L"Resetrep: Loaded %u factions\n", factions.size());
+		}
+	}
+	
+	map<wstring, uint> MiscCmds::Resetrep_Time_limits_load_map(string filename)
+	{
+		INI_Reader ini;
+		string factionpropfile = filename;
+		map<wstring, uint> tempmap;
+		if (ini.open(factionpropfile.c_str(), false))
+		{
+			while (ini.read_header())
+			{
+				if (ini.is_header("Playership"))
+				{
+					uint timelimit = 0;
+					wstring nickname = L"0";
+					while (ini.read_value())
+					{
+						if (ini.is_value("shipname"))
+						{
+							ini_get_wstring(ini, nickname);
+						}
+						else if (ini.is_value("timelimit"))
+						{
+							timelimit = ini.get_value_int(0);
+						}
+					}
+
+					tempmap[nickname] = timelimit;
+				}
+			}
+			ini.close();
+		}
+		return tempmap;
+	}
+
+	void MiscCmds::Resetrep_Time_limits_add(string filename, map<wstring, uint> tempmap)
+	{
+		string factionpropfile = filename;
```

Comment:
this doesn't seem to be necessary

Reply:
it looks like a function that rewrites/saves player time limits for all present charfilenames in some player folder

Reply:
see above

Reply:
understood the issue